### DISCUSSION TITLE
1.1.16 release and dependency update to System.Text.Json 7.0.3 leads to issues in in-process .NET 6 Azure Functions

### DISCUSSION THREAD

**Discussion by TobiasHeck** [microsoft/PowerPlatform-DataverseServiceClient#424] [Show and tell]
Hi all,

Just leaving this here, in case anyone experiences the same issue.

I recently tried updating the NuGet package to 1.1.16, which now has a dependency on System.Text.Json 7.0.3.
This lead to an Exception in one of our (Durable) Azure Functions:
`Exception while executing function: <OurFunctionName> Could not load file or assembly 'System.Text.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.`

For now, my solution was to simply downgrade to 1.1.14.

There are also other solutions, like using _FunctionsSkipCleanOutput or FunctionsPreservedDependencies ([see here with a good explanation from Bryan Knox](https://bryanknox.github.io/2022/07/15/functionsskipcleanoutput-and-functionspreserveddependencies.html)), or even migrating to the isolated worker model, which we most likely want to do in the near future as well.

We are running our Durable Functions using the following settings, etc.
.NET 6
In-Process
Function runtime 4

**Comment by MattB-msft** [comment]
Sorry about this issue, this is a byproduct of the critical dependencies bot's flagging assemblies as "compromised or unsafe" in some way that require us to move up the min dependencies.  

While it's not a great comfort to folks who experience this, 
We do document dependency changes in the release notes when we do this. 

Moving to discussions as Show and Tell. 
the Article attached is a good example of how to solve for this until Azure Functions takes the same change.
